scripts/travis-integration-tests.sh

```diff
@@ -51,6 +53,7 @@ kubectl() {
 # run_tests returns an error on failure
 run_tests
 exit_code=$?
+delete_k8s_cluster "${TEST_CONTEXT}"
```

Comment:
Can you implement delete_k8s_cluster for the other types (dind and minikube)? That way this will continue to work for those as well.

Reply:
It should work for other types, delete_k8s_cluster just call "cluster-down-xxx.sh" script if it's present.

Reply:
I missed that, so you are right, this will work ok. Thanks!